script/upload-node-checksums.py

```diff
@@ -33,27 +40,39 @@ def parse_args():
   parser = argparse.ArgumentParser(description='upload sumsha file')
   parser.add_argument('-v', '--version', help='Specify the version',
                       required=True)
+  parser.add_argument('-u', '--dist-url',
+                      help='Specify the dist url for downloading',
+                      required=False, default=DIST_URL)
+  parser.add_argument('-t', '--target-dir',
+                      help='Specify target dir of checksums',
+                      required=False)
   return parser.parse_args()
 
 
 def get_files_list(version):
   return [
-    'node-{0}.tar.gz'.format(version),
-    'iojs-{0}.tar.gz'.format(version),
-    'iojs-{0}-headers.tar.gz'.format(version),
-    'node.lib',
-    'x64/node.lib',
-    'win-x86/iojs.lib',
-    'win-x64/iojs.lib',
+    ['node-{0}.tar.gz'.format(version), True],
+    ['iojs-{0}.tar.gz'.format(version), True],
+    ['iojs-{0}-headers.tar.gz'.format(version), True],
+    ['node.lib', False],
+    ['x64/node.lib', False],
+    ['win-x86/iojs.lib', False],
+    ['win-x64/iojs.lib', False]
```

Comment:
Should've asked it earlier, but better late than never )
It's not obvious what `True/False` means here, can we use a dict like that:
```python
{'filename': 'node.lib', 'required': False}
```